nodeList.json

```
@@ -16,8 +16,12 @@
             "items": { "type": "string" },
             "description": "The first tag is considered the primary tag and should be unique"
           },
-          "color": { "type": "array", "items": { "type": "number" } },
-          "billboardColor": { "type": "array", "items": { "type": "number" } },
+          "color": { "type": "array", "items": {
+            "$ref": "./hexColor.json"
+          } },
+          "billboardColor": { "type": "array", "items": {
+            "$ref": "./hexColor.json"
+          } },
```

```suggestion
          "color": { "type": "array", "items": { "type": "number"  }},
          "billboardColor": { "type": "array", "items": { "type": "number"  }}
```
Those colors arent actually hex string just random numbers wich i will have to figure out how they work later